lib/ai/actions/message-actions.ts

```
@@ -105,8 +106,33 @@ export async function saveUserMessage({
     attachments: [],
   };
 
-  // Insert user message
-  const createdMessages = await insertMessages(supabase, [userMessageData]);
+  let createdMessages;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)